client/src/components/HomeText/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const HomeText = () => {
+  return (
+    <div className="d-flex flex-column align-items-center mt-2">
+      <h3 className="font-medium-high font-weight-normal text-white">
+        Don't know what to search?
+      </h3>
+      <p style={{ color: "#7a8c99" }}>Here's an offer you can't refuse</p>
```

Comment:
Sass could take care of styles and color variables, avoid spreading inline-style in the project.